causalpy/skl_experiments.py

```diff
@@ -11,721 +11,64 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
-"""Quasi-Experiment classes for OLS inference"""
+"""
+This module exists to maintain the old API where experiment classes were defined in the
+`causalpy.skl_experiments` namespace. They have moved to `causalpy.experiments` and
+this module is a thin wrapper around the new classes to maintain backwards
+compatibility. A deprication warning is delivered to the user. This module may be
```

Comment:
This isn't a british / american difference is it? 😄 
```suggestion
compatibility. A deprecation warning is delivered to the user. This module may be
```